recipes/libguarded/all/conanfile.py

```diff
@@ -0,0 +1,27 @@
+import os
+from conans import ConanFile, tools
+
+class LibguardedConan(ConanFile):
+    name = "libguarded"
```

Comment:
```suggestion
    name = "cs_libguarded"
```
I would prefer `cs_libguarded`. Both Cmake project name and github project are the same.